js/profcond.js

```diff
@@ -92,6 +92,9 @@
     if (state.css) {
       el.css(state.css);
     }
+    if (state.html) {
+      el.html(state.html);
```

Comment:
I feel like I should know this, but can you give me an example of how this is used?

Reply:
nvm. I see your comment at #3.